applications/polkadot-faucet.md

```diff
@@ -0,0 +1,217 @@
+# BlokkFaucet
+
+- **Team Name:** blokk by Apps with love AG
+- **Payment Details:**
+  - **DOT**: 13BXrKJVAamJwVYaKcHXvnUG24hpTtbAMbaPphTsEVsFi9Bh
+- **Level:** 1
+
+## Project Overview
+
+> Play, earn, and fuel the Polkadot ecosystem.
+
+This project builds a browser-based game secured by multiple OAuth providers and Cloudflare. Players can interact with the game and earn rewards through a faucet system. A secure .NET backend API manages the game logic, faucet transactions, and user data persistence using SQL Express databases. Transactions are facilitated through an external or self-hosted parachain node/API, integrating with the Polkadot ecosystem. The project will be periodically funded by automated or manual transfers from a sponsor wallet, ensuring a sustainable reward pool for players.
+
+The project utilizes a parachain node/API to process game transactions within the Polkadot ecosystem. This allows for secure and transparent in-game transactions, potentially using Kusama for testnet deployments or Polkadot for the mainnet version.
+
+We are passionate about creating a fun and engaging gaming experience while also introducing users to the power of blockchain technology through Polkadot. We believe this project has the potential to attract new users to the Polkadot ecosystem and increase parachain activity. We are interested in exploring innovative ways to leverage blockchain technology for gamification and user engagement.
+
+### Project Details
+
+Polkadot faucet which provides a browser game secured by several OAuth providers and Cloudflare. The game management and faucet transactions will be handled by a secured backend API written in .NET. The game, transactions, and keys will be persisted in SQL Express databases. Transactions between are done via an external or selfhosted parachain node/API. The faucet wallet will be filled through manual or automated transactions from the sponsor wallet after several time and faucet wallet standings.
+
+#### Figma Mockups
+
+![Fruit Ninja](https://github.com/user-attachments/assets/e42507ca-094d-4bd0-9ab5-72e74013c89d)
+![Mining](https://github.com/user-attachments/assets/f0789d45-4186-4981-9a69-f6c377f526ca)
+
+See [all screen mocks in Figma](https://www.figma.com/design/6C57eHkcYOL3gmOhg9Skv0/POW-Faucet-Game?node-id=1%3A27&t=QqnRK1Y0J9lNYL6o-1).
+
+You can access a [clickable Figma prototype](https://www.figma.com/proto/6C57eHkcYOL3gmOhg9Skv0/POW-Faucet-Game?page-id=1%3A27&node-id=1-28&viewport=410%2C293%2C0.17&t=4JhvjAeBiVyBlpeo-1&scaling=min-zoom&starting-point-node-id=1%3A28).
```

Comment:
What does 'Mining' refer to in this UI?

Reply:
It is not real mining, it is just a working title for this UI